book/getting-started/docker-setup.md

```diff
@@ -0,0 +1,55 @@
+# Setting Up Your Environment for Wallaroo in Docker
+
+To get you up and running quickly with Wallaroo, we've provided a Docker image which includes Wallaroo and related tools needed to run and modify a few example applications. We should warn that this Docker image was created with the intent of getting users started quickly with Wallaroo and is not intended to be a fully customizable development environment or suitable for production.
+
+## Installing Docker
+
+### MacOS
+
+There are [instructions](https://docs.docker.com/docker-for-mac/) for getting Docker up and running on MacOS on the [Docker website](https://docs.docker.com/docker-for-mac/).  We recommend the 'Standard' version of the 'Docker for Mac' package.
```

Comment:
impeartive voice is preferable here: "Follow the instructions on [the Docker website](...)..."

Reply:
some of these tone comments I won't address as they are the same across a few existing files. Don't want to change them all as part of this PR but also don't want to change it in a single place so we're carrying different tones throughput.